views/layout.erb

```diff
@@ -62,10 +62,10 @@
         <ul class="loginbar pull-right">
           <li><a href="/faqs">FAQs</a></li>
           <li class="devider"></li>
-          <li class="socialbutton slack"><a href="http://slack.fluentd.org"><img src="/images/slack_badge.png" width="55px" /></a></li>
+          <li class="socialbutton slack"><a href="//slack.fluentd.org"><img src="/images/slack_badge.png" width="55px" /></a></li>
```

Comment:
This is not missing protocol but making the resource protocol-dependent. See https://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http